Assignment03-Bootstrap/assets/scss/index.scss

```diff
@@ -0,0 +1,284 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif
```

Comment:
semicolon missing. Please add vs code actions to format on save

Reply:
Done.